syft/pkg/cataloger/java/archive_parser.go

```diff
@@ -273,7 +288,7 @@ func discoverPkgsFromOpener(virtualPath, pathWithinArchive string, archiveOpener
 	return nestedPkgs, nestedRelationships, nil
 }
 
-func pomPropertiesByParentPath(archivePath string, extractPaths []string, virtualPath string) (map[string]pkg.PomProperties, error) {
+func pomPropertiesByParentPath(archivePath, virtualPath string, extractPaths []string) (map[string]pkg.PomProperties, error) {
```

Comment:
I think this is a good change, but it makes a bit of unnecessary noise in the PR, would it be better to do this refactoring separately?